tests/loader.ts

```diff
@@ -0,0 +1,16 @@
+intern.registerLoader((options) => {
+	return intern.loadScript('node_modules/@dojo/loader/loader.js')
+		.then(() => intern.loadScript('./_build/src/util/amd.js'))
+		.then(() => {
+			(<any> require).config(shimAmdDependencies({
```

Comment:
I tried adding the typings to `require`. As soon as I import `@dojo/interfaces/loader`, the file turns into an actual module instead of just a plain function and the whole thing breaks down in agony.

Reply:
`(require as any)` and in a test case, the lack of typings is fine, though it would be good to put a comment in the code as to why you are casting.